waspc/data/Generator/templates/server/src/middleware/globalMiddleware.ts

```diff
@@ -6,6 +6,8 @@ import cors from 'cors'
 import helmet from 'helmet'
 
 import config from 'wasp/server/config'
+import type { MiddlewareConfig, MiddlewareConfigFn } from 'wasp/server/middleware'
```

Comment:
These types now moved to SDK, so here I import them from SDK and reexport them. They are used here, but also needed by other parts of server, and it made sense to reexport them.